docs/standard/library.md

```diff
@@ -1,105 +1,124 @@
 ---
-title: .NET Standard Library
-description: .NET Standard Library
-keywords: .NET, .NET Core
+title: .NET Standard | Microsoft Docs
+description: Learn what is .NET Standard, its versions and the .NET platforms supported.
+keywords: .NET Standard, PCL, .NET
 author: richlander
 ms.author: mairaw
-ms.date: 06/20/2016
+ms.date: 03/16/2017
 ms.topic: article
 ms.prod: .net
 ms.technology: dotnet-standard
 ms.devlang: dotnet
 ms.assetid: c044882c-af15-45f2-96d1-534557a5ee9b
 ---
 
-# .NET Standard Library
```

Comment:
I like the rename from ".NET Standard Library" to just ".NET Standard".

We should align the name with how we present it in the UX. We don't refer to ".NET Standard Library" anywhere in our UX anymore (yes, the package is still still `NetStandard.Library` but we basically go out of our way to hide that. Also, I've seen people getting confused about it because they don't know whether ".NET Standard Library" refers to their code or .NET Standard itself. Furthermore, the name should align with how we present it conceptually to our developers and we present it as a standard .NET platforms have to implement.